src/_includes/feature_lists/cloud.njk

```diff
@@ -1,224 +1,117 @@
+            id: "team-library",
+            label: "Team Library",
             values: [null, 'check', 'check'],
-            info: "<p>Use pipelines to ensure quality is delivered from your development stage, through staging, and your production environment</p>"
+            info: "<p>Setup standard nodes and flows that can be shared with all team members.</p>"
         }, {
             id: "sso",
             label: "Single Sign-On (SSO)",
             values: [null, null, 'check'],
             info: "<p>FlowFuse can be configured to run with your own Single Sign-On (SSO) provider. Allow users to access FlowFuse with a single set of login credentials, improving security, user experience, and save them time.</p>"
         }, {
-            label: "High Availibility",
-            values: [null, null, 'check'],
-            info: "<p>Leverage horizontal scaling for reliable and scalible processing of your data through Node-RED.</p>"
-        }, {
-            label: "Custom Node Catalog",
-            values: [null, 'time', 'time']
-        }, {
-            label: "Private npm Repo Support",
-            values: [null, 'time', 'time']
+            id: "project-nodes",
+            label: "Seamless Data Stream",
+            values: [null, 'check', 'check'],
+            info: "<p><a href=\"https://github.com/flowforge/flowforge-nr-project-nodes\" target=\"blank\">FlowForge Project Nodes</a> enable the passing of data and messages between your Node-RED projects, included with all FlowForge Projects by default.</p>"
         }]
     }, {
-        label: "Limits",
+        label: "Support & SLA",
+        hosting: "cloud",
         rows: [{
-            label: "Maximum Teams",
-            values: [50, "Unlimited", "Unlimited"]
+            label: "Committed SLA",
+            values: ['None', '99.5%', 'Up to 99.9%']
         }, {
-            label: "Maximum Local Instances",
-            values: [50, "Unlimited", "Unlimited"]
+            label: "FlowForge Support",
+            values: ['<a href="https://community.flowfuse.com/">Community Forum</a>', 'Next business day', '4 hours']
         }, {
-            label: "Maximum Remote Instances",
-            values: [50, "Unlimited", "Unlimited"]
-    }]
+            label: "Node-RED Support",
+            values: ['<a href="https://discourse.nodered.org/">Community Forum</a>', 'Next business day', '4 hours']
+        }]
     }, {
-        label: "Support",
+        label: "Subscription",
+        hosting: "cloud",
         rows: [{
-            label: "Community Support",
-            values: ['check', 'check', 'check']
-        }, {
-            label: "Ticket-Based Support",
-            values: [null, 'SLA: Next Business Day', 'SLA: 4 hours']
-        }, {
-            label: "Long-Term Support",
-            values: [null, 'check', 'check']
-        }]
+            label: "Price",
+            values: ['$15/month', 'Starting at $25/instance', 'Contact Sales']
```

Comment:
```suggestion
            values: ['$9/month', '$19/instance <br> $15/device, 'Contact Sales']
```

Reply:
Discussion result with @ZJvandeWeg and @zackwasli

Reply:
And in this section, are we displaying two prices in the team plan?
<img width="1008" alt="Screenshot 2023-08-29 at 09 44 30" src="https://github.com/flowforge/website/assets/129537638/13fb96cc-39a3-409d-af3f-54c27c35d183">